internal/tiltfile/config/resources.go

```diff
@@ -100,10 +104,11 @@ func match(manifests []model.Manifest, requestedManifests []model.ManifestName)
 		addManifestAndDeps(manifestsToRun, manifestsByName, m)
 	}
 
-	var result []model.Manifest
+	var result []model.ManifestName
 	for _, m := range manifests {
-		if manifestsToRun[m.Name] {
-			result = append(result, m)
+		// Default to including UnresourcedYAML ("Uncategorized") to match historical behavior.
```

Comment:
I'm not sure this is the right long-term behavior but didn't want to unnecessarily expand the scope of this change.